_guides/includes/create-app.adoc

```diff
@@ -0,0 +1,60 @@
+[role=primary]
+.CLI - Maven project
```

Comment:
This wraps weirdly on a phone FYI...

Reply:
@kenyee thanks for the heads up. @insectengine will tweak this thing to make it prettier and more compatible.